### DISCUSSION TITLE
Mirror Image

### DISCUSSION THREAD

**Discussion by gkr321** [Bodmer/TFT_eSPI#1811] [Compatible displays and setup files]
I bought a couple of these displays just to learn how to use them. https://www.aliexpress.com/item/32914468153.html?spm=a2g0o.order_list.0.0.21ef1802BMNl9g
It's the .96 inch with the ST7735S chip
It also has a blue tab on the screen protector.
I've hooked one up and ran the TFT_Meter5 demo. It's working except that it is a mirror image.
How do i fix this?

**Comment by bionicbone** [comment]
I've recently have seen a long thread on this chip, I remember the chip number printed on the board was not the actual chip used.

Found it, check this thread.

https://github.com/Bodmer/TFT_eSPI/discussions/1310

**Reply by gkr321** [reply]
thanks
